winui/Rating/Precision.md

```diff
@@ -10,3 +10,86 @@ documentation: ug
 # Precision in WinUI Rating
 
 This section explains the precision features available in the WinUI Rating control.
+
+## Rating Control using the Precision property
+
+The Rating control Precision can be customized using the `Precision` Property. It have three types Full, Half and Exact. The default value is "Full".  
+
+## Full
+
```

Comment:
@Priyanka-Vijayasankar Rating value can be provided with whole precision.

Reply:
Added the above mentioned content.